### ISSUE TITLE
server and cluster setup

### ISSUE THREAD

**Issue by kizariel** [arkmanager/ark-server-tools#973]
Hi, i am trying to setup 2 clustered server, one install in each directory with the same UID
-home/user/island
-home/user/ragnarok
but when i install server manager it put the file in /user and no island or ragnarok.
(on debian 8)
Is there something to do ?
I am sorry for my really bad english.

**Comment by Justinfact** [comment]
Did you define
`arkserverroot="/home/user/ragnarok"
arkserverroot="/home/user/island"`

...in the individual server instance configuration files (/etc/arkmanager/instances/<map>.cfg) ? That's defining where it installs the files.

**Comment by kizariel** [comment]
ty for your repply, with 
`arkserverroot="/home/user/(ragnarok or island) `
can i have 2 full install and run one instance only for each install?

Actually i have 2 different users with one install and one instance for each and want the same 
thing under only one user tu setup my cluster.

**Comment by Justinfact** [comment]
Assuming you installed SteamCMD correctly somewhere before arkmanager, your new issue is likely with /etc/arkmanager/arkmanager.cfg not containing the correct SteamCMD config. Here's an extract of that section from my arkmanager.cfg (my Linux user for arkmanager is called "steam"):

```
# config SteamCMD
steamcmdroot="/home/steam/steamcmd"
steamcmdexec="steamcmd.sh"
steamcmd_appinfocache="/home/steam/Steam/appcache/appinfo.vdf"
steamcmd_workshoplog="/home/steam/Steam/logs/workshop_log.txt"
steamlogin="anonymous"
steamcmd_user="steam"
```

You can define all of that in the arkmanager.cfg, assuming you want to use the same SteamCMD config for both instances.

For the second issue, is there an arkmanager file in your /usr/local/bin and is the arkmanager.cfg configured with:
`install_bindir="/usr/local/bin"`

**Comment by kizariel** [comment]
I just found that the user was in [root] group .... so the steam cmd error is fix (remove my user from the group).
For the second error actually i have `install_bindir="/home/user/bin"` 
so i need to change it  in : `install_bindir="/user/bin"` ?
Sorry again i am learning at the same time necause it's my first time with a server under linux ^^".

**Comment by klightspeed** [comment]
`install_*` is used by the updater to locate the installed arkmanager files.  Those settings are not referenced by the ARK install etc.

For the second error, you can add `~/bin` to your PATH - it looks like you performed a user-local install, which puts arkmanager into `~/bin`, its config into `~/.arkmanager.cfg`, and the instance configs into `~/.config/arkmanager/instances`

**Comment by kizariel** [comment]
I use `curl -sL http://git.io/vtf5N | bash -s -- --me`  with `--perform-user-install`
is that's why i need to use `./bin/arkmanager install` instead of `arkmanager install `?

**Comment by kizariel** [comment]
mhhhh i have a new strange problem, i have user cronjob to add a daily restart, but the instance do it in loop.
From 6h to 7h serv restart is looping:

`2018-08-17 06:04:04: [ragnarok] Stopping server; reason: restart
2018-08-17 06:04:08: [ragnarok] The server has been stopped
2018-08-17 06:04:08: stop
2018-08-17 06:04:09: [ragnarok] Checking for updates before starting
2018-08-17 06:04:09: [ragnarok] Checking for update; PID: 30459
2018-08-17 06:04:11: [ragnarok] Start aborted due to running update - pid: 30459
2018-08-17 06:04:16: [ragnarok] Your server is already up to date! The most recent version is 3029740.
2018-08-17 06:04:16: [ragnarok] The server is starting...
2018-08-17 06:04:16: [ragnarok] The server is now running, and should be up within 10 minutes
2018-08-17 06:05:29: [ragnarok] Stopping server; reason: restart
2018-08-17 06:05:32: [ragnarok] The server has been stopped
2018-08-17 06:05:32: stop`

**Comment by klightspeed** [comment]
It sounds like you have `* 6 * * * arkmanager @instance restart`, which will execute the command every minute between 6:00 and 7:00.
Try using `0 6 * * * ...`

**Comment by kizariel** [comment]
omg i am so stupîd =/ really sorry
